Remove unused RenderListBox::valueChanged

Ahmad-S792 opened the pull request:
#### c8fd7771b683835ba4469a6dec8d0b271f14270b
<pre>
Remove unused RenderListBox::valueChanged

Remove unused RenderListBox::valueChanged
<a href="https://bugs.webkit.org/show_bug.cgi?id=250638">https://bugs.webkit.org/show_bug.cgi?id=250638</a>

Reviewed by Alan Baradlay.

This patch is to remove unused &quot;valueChanged&quot; function and related in header.

Merge - <a href="https://src.chromium.org/viewvc/blink?view=rev&amp">https://src.chromium.org/viewvc/blink?view=rev&amp</a>;revision=169543

* Source/WebCore/rendering/RenderListBox.cpp:
(RenderListBox::valueChanged): Removed
* Source/WebCore/rendering/RenderListBox.h: Remove &quot;valueChanged&quot;

Canonical link: <a href="https://commits.webkit.org/258955@main">https://commits.webkit.org/258955@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/805ebeeb97ed4e6ff5676ac286d1fa45bbf20389

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/103456 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/12575 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/36419 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/112692 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/172901 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/107409 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/13607 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/3479 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/95709 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/111878 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/109229 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/10466 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/93548 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/38188 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/92287 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/25128 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/79843 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/5964 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/26531 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/6140 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/3063 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/12122 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/46040 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6154 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/7896 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

alanbaradlay commented on Source/WebCore/rendering/RenderListBox.h:
```diff
@@ -170,7 +171,6 @@ class RenderListBox final : public RenderBlockFlow, public ScrollableArea {
     void computeFirstIndexesVisibleInPaddingTopBottomAreas();
 
     LayoutUnit itemHeight() const;
-    void valueChanged(unsigned listIndex);
```
It looks like it was never used.

Ahmad-S792 replied:
Yes - it was dead-code. So cleaning-up. :-)

Ahmad-S792 committed:
Remove unused RenderListBox::valueChanged

Remove unused RenderListBox::valueChanged
https://bugs.webkit.org/show_bug.cgi?id=250638

Reviewed by Alan Baradlay.

This patch is to remove unused "valueChanged" function and related in header.

Merge - https://src.chromium.org/viewvc/blink?view=rev&revision=169543

* Source/WebCore/rendering/RenderListBox.cpp:
(RenderListBox::valueChanged): Removed
* Source/WebCore/rendering/RenderListBox.h: Remove "valueChanged"

Canonical link: https://commits.webkit.org/258955@main
